buildtest/tools/menu.py

```diff
@@ -367,40 +363,3 @@ def show_menu(self):
         parser_schema = subparsers_show.add_parser("schema", help="Display YAML schema")
         parser_schema.set_defaults(func=show_schema_layout)
         parser_show.set_defaults(func=func_show_subcmd)
-
-    def testconfigs_menu(self):
```

Comment:
Makes sense to remove ``buildtest testconfigs`` for now as it relates to how configuration name is set.